docs/windows.md

```diff
@@ -45,7 +45,9 @@ and daemons are pre-installed on the Windows Nodes in the demo.
   updates.
 * Deploy a Linux-based Kubernetes cluster.
 * Install [Hyper-V](https://docs.microsoft.com/en-us/windows-server/virtualization/hyper-v/get-started/install-the-hyper-v-role-on-windows-server)
-  with management tools.
+  with management tools. If the virtualization capabilities on your Nodes
```

Comment:
How about "If you Nodes do not have the virtualization capabilities required by Hyper-V"?